preprocessing_data.py

```diff
@@ -0,0 +1,17 @@
+import cv2
+import glob
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+import splitfolders
+for name in ['COVID',  'Normal']:
+    for file in glob.glob(f"COVID-19_Radiography_Dataset/{name}/*.png"):
```

Comment:
Just a suggestion. Change this to be generic as well?

Reply:
done